README.md

```diff
@@ -1,9 +1,67 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, HTML/CSS, Docker
+
+**Competent:** Python, C/C++/C#, Spring, JUnit, Flutter, PHP, JavaScript, Maven
+
+**Familiar:** Subversion, Jenkins, Node.js, React.js, Eclipse
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed)
+
+
+**Cluster Monitor** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/linux_sql)]: Utilizes Liunx Bash, Postgresql, and Docker to build a monitoring agent that collects data on Linux machine specifications and performance in a cluster.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/core_java)]:
+      
+  - Twitter App: Incorporates Spring Boot, Maven, Docker, and leverages the Twitter API in order to deliver a command line interface to create, read, and delete tweets.
+  - JDBC App: Sets up data manipulation to a retail database using Java, Maven, Docker and Postgresql.
+  - Grep App: Pipelines information on a target directory for files and lines via Java 8 Streams, along with Docker and Maven, in order to find a given regular expression.
+
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/springboot)]: Demonstrates a trading system Proof of Concept for stock traders. Leverages Spring Boot, Maven, Docker, Postgresql, and the IEX API.
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/python_data_analytics)]: Analyzes data incoming in sql files for marketing purposes using Docker, Python, Postgresql, and Jupyter Notebook.
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/hadoop)]: To be implemented.
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/spark)]: To be implemented
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HabibMohamed/tree/master/cloud_devops)]: To be implemented
+
+
+## Highlighted Projects
+**TSVtoMySQL** [[GitHub](https://github.com/abdiclear/TSVtoMySQL)]: Takes in TSV/CSV files and inserts the data into an existing MySQL database.
+
+**Department Wars** [[GitHub](https://github.com/abdiclear/department_wars)]: A small interactive React.js app that plays around with the concept of Routes and Redux.
+
+
+## Professional Experiences
+
+**Data Engineer, Jarvis (2020-present)**: Progressed through various projects and requirements using Java 8, Spring Boot, Linux, Git, Python, Docker, and other software.
+
+**Research Assistant, Department of Computer Science, University of Windsor (2019-2020)**: Manipulated gigabytes of data in TSV/CSV format into being inserted in MySQL databases using PHP, JavaScript, and HTML/CSS/BootStrap.
```

Comment:
talk about team collaboration, leadership, and other soft skills

Reply:
please write more about team collaboration, leadership, and other soft skills for all professional exp.